index.html

```diff
@@ -59,6 +59,38 @@
         </div>
         </div>
     </nav>
+    </nav>
+
+    <section class="hero flex column">
+        <div class="flex column">
+            <h1>Unsplash</h1>
+            <h2>The internet's source for visuals </h2>
+            <h2> Powered by creators everywhere. </h2>
+            <form class="relative text-input-with-icon hero_item--desktop-only">
+                <input class="text-input text-input-with-icon search-input" placeholder=" Search" type="text"
```

Comment:
You have the same class - `search-input` as the Header input.

This means the same responsive rules apply, thus the input is 20vw on smaller screens:
<img width="592" alt="image" src="https://user-images.githubusercontent.com/11327761/204128889-e456b419-665b-438b-abd0-a78095059fe6.png">